### DISCUSSION TITLE
baize/developer/interface/create/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4242] [Announcements]
# baize/developer/interface/create/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/baize/developer/interface/create/

**Comment by zgfh** [comment]
curl -X POST localhost:8000/v2/models/vllm_model/generate -d '{"text_input": "What is Triton Inference Server?", "parameters": {"stream": false, "temperature": 0}}'

**Comment by ModetaNiu** [comment]
fixed, see https://github.com/DaoCloud/DaoCloud-docs/pull/4548
